conductor-client/src/main/kotlin/com/openlattice/organizations/WarehouseService.kt

```diff
@@ -0,0 +1,62 @@
+package com.openlattice.organizations
+
+import com.google.common.base.Preconditions
+import com.hazelcast.core.HazelcastInstance
+import com.openlattice.authorization.AuthorizationManager
+import com.openlattice.authorization.AclKey
+import com.openlattice.authorization.securable.SecurableObjectType
+import com.openlattice.authorization.HazelcastAclKeyReservationService
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.organizations.JdbcConnectionParameters
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+/**
+ * @author Andrew Carter andrew@openlattice.com
+ */
+
+@Service
+class WarehouseService(
```

Comment:
I think it would be more consistent if it was plural ... warehouses mapstore, warehouses servce, etc.

```suggestion
class WarehousesService(
```